specification/monitor/resource-manager/Microsoft.Insights/preview/2017-05-01-preview/metrics_API.json

```diff
@@ -77,6 +77,9 @@
           },
           {
             "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/MetricNamespaceParameter"
```

Comment:
I don't think the 2017-05-01-preview version of the API supported the metricNamespace parameter